package.json

```diff
@@ -118,6 +118,7 @@
     "cosmiconfig": "^6.0.0",
     "debug": "^4.1.1",
     "execall": "^2.0.0",
+    "fast-glob": "^3.2.4",
```

Comment:
This is already a dependency of `globby`, so it's not really a new dependency here.

Reply:
Just note - prettier and many other tools remove `globby` and use `fast-glob` directly, maybe we can do same here

Reply:
Oh interesting! That could be a good follow-up PR. 

It would need to be a breaking change, I think.  As `globbyOptions` is a config option for stylelint. `globby` has a couple of options that aren't supported in `fast-glob`. Though I'd guess few people are using any of the globby-only option..